kf_hungarian_tracker/kf_hungarian_tracker/kf_hungarian_node.py

```diff
@@ -28,25 +31,28 @@ class KFHungarianTracker(Node):
         detection_sub: subscrib detection result from detection node
         tracker_obstacle_pub: publish tracking obstacles with ObstacleArray
         tracker_pose_pub: publish tracking obstacles with PoseArray, for rviz visualization
-    '''
+    """
 
     def __init__(self):
-        '''initialize attributes and setup subscriber and publisher'''
+        """initialize attributes and setup subscriber and publisher"""
 
-        super().__init__('kf_hungarian_node')
+        super().__init__("kf_hungarian_node")
         self.declare_parameters(
-            namespace='',
+            namespace="",
             parameters=[
-                ('global_frame', "camera_link"),
-                ('process_noise_cov', [2., 2., 0.5]),
-                ('top_down', False),
-                ('death_threshold', 3),
-                ('measurement_noise_cov', [1., 1., 1.]),
-                ('error_cov_post', [1., 1., 1., 10., 10., 10.]),
-                ('vel_filter', [0.1, 2.0]),
-                ('height_filter', [-2.0, 2.0]),
-                ('cost_filter', 1.0)
-            ])
+                ("global_frame", "camera_link"),
+                ("process_noise_cov", [2.0, 2.0, 0.5]),
+                ("top_down", False),
+                ("death_threshold", 3),
+                ("measurement_noise_cov", [1.0, 1.0, 1.0]),
+                ("error_cov_post", [1.0, 1.0, 1.0, 10.0, 10.0, 10.0]),
+                ("vel_filter", [0.1, 2.0]),
+                ("height_filter", [-2.0, 2.0]),
+                ("cost_filter", 1.0),
+                ("transform_to_global_frame", False),
+                ("infer_orientation_from_velocity", False),
```

Comment:
Default true?

Reply:
fixed in 3299d8e7a0fef70b9dd6a77d7050eb4e61aed4ab